test/testinput/increment.yml

```
@@ -1,5 +1,21 @@
 geometry:
-  # insert other model specific things here
+  grid: S360x180
+  domain:
+    type: global
+    west: -180
 
-inc variables: [var1, var2]
-test date: 2018-04-15T00:00:00Z
+read_from_file: 1
+
+# do not know what it is yet, most likely will set up wrongly
+dirac:
+  date: 1985-01-01T12:00:00Z
+  ixdir: [20]
+  iydir: [10]
+
+inc variables: [x]
+
+increment test:
+  date: 1985-01-01T12:00:00Z
+  tolerance: 1e-7
+# filename: Data/19850101_regridded_sst.nc
+# added variables: [x]
```

```suggestion
inc variables: [sea_surface_temperature]

increment test:
  date: 1985-01-01T12:00:00Z
  tolerance: 1e-7
```

`dirac` will be used with a different test, not here.
The increment variables are `sea_surface_temperature`, not `x`, the fact the tests are still passing despite this means there is probably a bug somewhere in your code... we'll deal with that later